cypress/integration/content/headTags_spec.js

```diff
@@ -13,9 +13,9 @@ describe("Head Tags", () => {
 
     cy.get("[data-cy=contentNavButton]").click();
 
-    cy.get("[data-cy=head]").click();
+    cy.get("[data-cy=head]").click({ force: true });
```

Comment:
contentNav wont close  event with `cy.get("[data-cy=contentNavButton]").click();` and is blocking the click of Head Tag button that's why I adde force true